wolf/ml/referee_ocr/image_processor.h

```diff
@@ -0,0 +1,147 @@
+#ifndef ImageProcessor_H
+#define ImageProcessor_H
+
+#include <opencv2/opencv.hpp>
+#include <string>
+#include <vector>
```

Comment:
because this header contains C++ headers such as `<vector>`, so please rename the file extension to  .hpp